test/ui/app/App.test.js

```diff
@@ -106,6 +107,14 @@ const mountWithRoute = route => mountWithIntl((
   </MemoryRouter>
 ));
 
+// A helper to update wrapper - used for waiting for lazy loaded pages
+const waitForComponentToPaint = async (wrapper) => {
+  await act(async () => {
+    await new Promise(resolve => setTimeout(resolve));
+    wrapper.update();
+  });
+};
+
```

Comment:
In react-testing-library, there is a method for this case https://testing-library.com/docs/dom-testing-library/api-async/#waitfor have you checked? This could help to prevent the creation of this helper.